proof_of_stake/src/types.rs

```diff
@@ -328,6 +328,45 @@ pub struct GenesisValidator {
     pub commission_rate: Dec,
     /// Maximum change in commission rate permitted per epoch
     pub max_commission_rate_change: Dec,
+    /// Validator metadata
+    pub metadata: ValidatorMetaData,
+}
+
+/// Validator metadata
+#[derive(
+    Clone,
+    Debug,
+    BorshSerialize,
+    BorshSchema,
+    BorshDeserialize,
+    Eq,
+    Ord,
+    PartialOrd,
+    PartialEq,
+)]
+pub struct ValidatorMetaData {
+    /// Validator's email
+    pub email: String,
+    /// Validator description
+    pub description: Option<String>,
+    /// Validator website
+    pub website: Option<String>,
+    /// Validator alias
+    pub alias: Option<String>,
+    /// Validator's discord handle
+    pub discord_handle: Option<String>,
+}
+
```

Comment:
```suggestion

#[cfg(any(test, feature = "testing"))]
```
(to avoid using the default email somewhere in non-test code)